kerchunk/hdf.py

```diff
@@ -496,6 +516,8 @@ def _translator(
                         if h5obj.fletcher32:
                             logging.info("Discarding fletcher32 checksum")
                             v["size"] -= 4
+                        key =  str.removeprefix(h5obj.name, "/") + "/" + ".".join(map(str, k))
```

Comment:
This is the same as what `_chunk_key` did? Maybe make it a function with a comment saying it's a copy/reimplementation.

By the way, is `h5obj.name` not actually a string, so you could have done `h5obj.name.removeprefix()`?